lib/oli_web/live/objectives/listing.ex

```diff
@@ -0,0 +1,77 @@
+defmodule OliWeb.ObjectivesLive.Listing do
+  use Surface.Component
+
+  alias OliWeb.ObjectivesLive.Actions
+
+  prop rows, :list, required: true
+  prop selected, :string, required: true
+
+  def render(assigns) do
+    ~F"""
+      <div id="accordion" class="accordion p-3">
+        {#for {item, index} <- Enum.with_index(@rows)}
+          <div class="card border border-light mb-2">
+            <div class="card-header d-flex justify-content-between p-2" id={"heading#{index}"}>
+              <button
+                class="btn w-75 text-left"
+                data-toggle="collapse"
+                data-target={"#collapse#{index}"}
+                aria-expanded="true"
+                aria-controls={"collapse#{index}"}
+                :on-click="set_selected"
+                :values={slug: item.slug}>
+                {item.title}
+              </button>
+              <div class="d-flex flex-column font-weight-light small bg-secondary p-2 rounded mr-2">
+                <div><i class="fa fa-cubes c0183 mr-1"></i>Sub-Objectives {item.sub_objectives_count}</div>
+                <div><i class="far fa-file c0183 mr-1"></i>Pages {item.page_attachments_count}</div>
+                <div><i class="fa fa-list mr-1"></i>Activities {item.activity_attachments_count}</div>
+              </div>
+            </div>
+
+            <div id={"collapse#{index}"} class={"collapse" <> if item.slug == @selected, do: " show", else: ""} aria-labelledby={"heading#{index}"} data-parent="#accordion">
+              <div class="card-body">
+                <div class="mb-3">
+                  <u>Sub-Objectives</u>
```

Comment:
We could also make a collapse accordion here and just show `Sub-objectives` or `Pages` based on what the user wants to see (instead of showing all the info at once)